aspnetcore/test/integration-tests.md

```diff
@@ -295,12 +295,9 @@ Call the [UseSolutionRelativeContentRoot](/dotnet/api/microsoft.aspnetcore.testh
    {
        protected override void ConfigureWebHost(IWebHostBuilder builder)
        {
-           builder.ConfigureServices(services =>
-           {
-               builder.UseSolutionRelativeContentRoot("<SOLUTION-RELATIVE-PATH>");
+           builder.UseSolutionRelativeContentRoot("<SOLUTION-RELATIVE-PATH>");
```

Comment:
I'm confused, why are recommending this in the first place? The WAF base class should take care of this automatically.

Reply:
We say this today ...

> In most cases, it isn't necessary to explicitly set the app content root, as the search logic usually finds the correct content root at runtime. In special scenarios where the content root isn't found using the built-in search algorithm, the app content root can be specified explicitly or by using custom logic. To set the app content root in those scenarios, call the `UseSolutionRelativeContentRoot` extension method ...

Would you like to drop all of that + the code example?

Reply:
I think so.

Reply:
Thanks @javiercn ... I hope to be free-er-ish later today, and I'll get this squared away.